packages/ag-charts-enterprise/src/features/navigator/navigatorDOMProxy.ts

```diff
@@ -120,14 +121,20 @@ export class NavigatorDOMProxy {
         return { offsetX: this.dragStartX + event.originDeltaX };
     }
 
-    private onDragStart(
-        slider: _ModuleSupport.SliderWidget,
-        event: _ModuleSupport.DragWidgetEvent<'drag-start'>,
-        key: NavigatorButtonType
-    ) {
+    private moveToFront(index: number) {
+        if (index === 1) return; // ignore pan-handle
+
+        const frontSlider = this.sliders[index];
+        const otherSlider = this.sliders[[2, NaN, 0][index]];
```

Comment:
This might be easier to follow than the double index lookup

```suggestion
    private moveToFront(index: 0 | 1 | 2) {
        if (index === 1) return; // ignore pan-handle

        const frontSlider = this.sliders[index];
        const otherSlider = this.sliders[2 - index];
```

Reply:
Thanks for the feedback. I've updated the PR.